.github/workflows/jira-issue-create.yml

```diff
@@ -0,0 +1,34 @@
+name: Jira Issue Creator
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    environment: Jira
+    name: SDK Bot Jira Issue Creation
+    steps:
+      - name: Login
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create issue
+        id: create
+        uses: atlassian/gajira-create@master
+        with:
+          project: ${{ secrets.JIRA_PROJECT }}
+          issuetype: Task
+          summary: |
+            [SDK - Node] ${{ github.event.issue.title }}
+          description: |
+            ${{ github.event.issue.url}}
+          fields: '{"customfield_10006": 1039, "customfield_11481": {"value": "Data Management"}, "customfield_11200":
+            {"value": "SDK"}}' # sprint, pillar, pod
```

Comment:
nit: say which sprint this is?

Reply:
this is probably overkill but this could potentially be secrets/env vars since they are magick-ish values

Reply:
will do that